app/assets/stylesheets/profiles.scss

```diff
@@ -0,0 +1,21 @@
+// Place all the styles related to the profiles controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.profile-page {
+  width: 100%;
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
+}
+
+.cover-placeholder {
+  background: 50% no-repeat #fafafa;
```

Comment:
Color literals like `#fafafa` should only be used in variable declarations; they should be referred to via variable everywhere else.